js/pathfinding.js

```diff
@@ -69,7 +115,14 @@ function getCachedNodes(token) {
 		cacheData.tokenWidth = token.data.width;
 		cacheData.tokenHeight = token.data.height;
 	} else if (canvas.grid.isHex && game.modules.get("hex-size-support")?.active) {
-		cacheData.tokenSize = token.document.data.flags["hex-size-support", "borderSize"];
+		const hexSizeSupportFlags = token.document.data.flags["hex-size-support"];
+		if (hexSizeSupportFlags) {
+			cacheData.tokenSize = hexSizeSupportFlags["borderSize"];
+			if (cacheData.tokenSize && cacheData.tokenSize % 2 === 0) {
+				cacheData.altSnap = hexSizeSupportFlags["altSnapping"];
+				cacheData.evenSnap = hexSizeSupportFlags["evenSnap"];
```

Comment:
I don't think that `evenSnap` is utilized anywhere in Drag Ruler, so we can safely ignore them here. However, storing the alternative orientation flag might be important. It can be accessed using `CONFIG.hexSizeSupport.getAltOrientationFlag(token)`.